### PR TITLE
docs: update issue templates

### LINKED ISSUE #1381: Refresh issue creation template
The community checklist at https://github.com/Azure/aks-engine/community thinks the project needs issue templates, so let's add one for features and one for bug reports.

### PR DESCRIPTION
**Pull request by mboersma**
**Reason for Change**:
Adds separate issue and feature [issue templates](https://help.github.com/en/articles/about-issue-and-pull-request-templates) initially generated by the GH wizard. These should be more targeted and less verbose, but I would love feedback on how to make them more user-friendly.

**Issue Fixed**:
Fixes #1381

**Requirements**:
- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+---
+
+**Describe the request**
+
+**Describe the solution you'd like**
+
+**Describe alternatives you've considered**
+
+**Additional context**
```
Comment (CecileRobertMichon) [inline .github/ISSUE_TEMPLATE/feature_request.md]: I'd add something like "why is this needed" , ie. convince us that we want this

**Comment by CecileRobertMichon** [comment]
Kubernetes also has failing test and support. It's pretty cool that we can customize these to add specific categories. Do you think having a support or question category is overkill? I guess for those people can use the "regular issue" button.

https://github.com/kubernetes/kubernetes/tree/master/.github/ISSUE_TEMPLATE

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: Bug report
+about: Create a report to help AKS Engine improve
+title: ''
+labels: ''
```
Comment (CecileRobertMichon) [inline .github/ISSUE_TEMPLATE/bug_report.md]: maybe we can use this to add the "bug" label automatically?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+name: Feature request
+about: Suggest an idea for AKS Engine
+title: ''
+labels: ''
```
Comment (CecileRobertMichon) [inline .github/ISSUE_TEMPLATE/feature_request.md]: and here the "enhancement" label

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -1,42 +0,0 @@
-<!-- Thanks for filing an issue! Before hitting the button, please answer these questions. It's helpful to search the existing GitHub issues first. It's likely that another user has already reported the issue you're facing, or it's a known issue that we're already aware of-->
-
-**Is this a request for help?**:
```
Comment (CecileRobertMichon) [inline .github/ISSUE_TEMPLATE.md]: looking forward to getting rid of this... I don't think I've ever seen someone answer "no" to "Is this a request for help?"

**Comment by mboersma** [comment]
> Do you think having a support or question category is overkill? 

Not necessarily—I almost added a "question" template when creating this. Maybe for now we use "bug," "feature," and the "regular issue" link and see how it goes?

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (mboersma) [reply .github/ISSUE_TEMPLATE/feature_request.md]: That's important, so I added it as the second section in feature request template.
